README.md

```diff
@@ -46,13 +46,26 @@ Options:
    --version                   print version and exit
 ```
 
-This passes the source of all passed through the transform module specified
-with `-t` or `--transform` (defaults to `transform.js` in the current
-directory). The next section explains the structure of the transform module.
+This passes the source of all passed through the transform modules specified
+with `-t` or `--transform` (they default to the current directory). The next
+section explains how transform modules are located, and the section after that
+explains the structure of transform modules.
+
+## Transform location algorithm
+The default transform is the current directory if no `--transform` flag is given.
+
+1. If URL, return file at URL
+2. If file, return file
+3. If directory
+  1. If contains directory named `transforms`, return files in that directory
+  2. If contains file named `transform.js`, return file
+  3. Else, return files in directory
+4. If npm package, find its directory (with `require.resolve()`) and go to step 3
```

Comment:
I believe it's based off the module paths (which `require()` and `require.resolve()` use internally), so that should be right. I assumed this would be obvious enough to users since module paths for local packages are separated from global packages. There are other Node tools (including Karma and Grunt) that rely on plugin modules also being installed locally/globally. However, I could add a note about this if you don't think it would be obvious.